DeepSpeech.py

```diff
@@ -787,7 +787,7 @@ def export():
     outputs['metadata_alphabet'] = tf.constant([Config.alphabet.serialize()], name='metadata_alphabet')
 
     if FLAGS.export_language:
-        outputs['metadata_language'] = tf.constant([FLAGS.export_language.encode('ascii')], name='metadata_language')
+        outputs['metadata_language'] = tf.constant([FLAGS.export_language.encode('utf-8')], name='metadata_language')
```

Comment:
Why?

Reply:
Because french ;) « Français »

Reply:
Ah, I was thinking of language codes like fr-FR.

Reply:
Yeah, that's open to discussion, I'd like to expose a nice name in the JSON. If we keep `metadata_language` for language code, then we need a new flag in `util/flags.py`. Your call :)